movai_core_shared/envvars.py

```diff
@@ -150,6 +150,7 @@
     "Annotation",
     "Layout",
     "GraphicScene",
+    "Translation",
```

Comment:
Both `REST_SCOPES` and `SCOPES_TO_TRACK` (and maybe others, haven't checked) shouldn't really be in `envvars.py`, since they're not read from the environment. We should either move them to `consts.py`, or just move them both to the Backend, since they're not really _shared_ values (which should be the purpose of this repo - stuff shared with the whole platform), they're only useful for the Backend.

Reply:
Moved both to dal